build/rhel.Dockerfile

```diff
@@ -1,38 +1,43 @@
-# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhel8/go-toolset
+# Build the manager binary
 FROM registry.redhat.io/rhel8/go-toolset:1.13.4-27 as builder
 
 ENV GOPATH=/go/
-
 USER root
 
-WORKDIR /devworkspace-operator
-
-# Populate the module cache based on the go.{mod,sum} files.
-COPY go.mod .
-COPY go.sum .
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
 RUN go mod download
 
-# copy the rest of the sources code
+# Copy the go source
 COPY . .
+
 # compile workspace controller binaries
 RUN CGO_ENABLED=0 GOOS=linux go build \
```

Comment:
Should we also include GOARCH=amd64 GO111MODULE=on here as well? and same with line 26?

Reply:
Good catch; the vars were set by kubebuilder so I'm not 100% clear on if they're actually needed though. I imagine at some point we might have to undo the change to support multi-platform builds.